### PR TITLE
add tweaks

### PR DESCRIPTION
**Pull request by jshawl**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,12 +5,12 @@ On thursday and friday of the coming week(28 May and 29 May), we'll be having Wh
...
 - To do research on something and give a presentation to a class
-- Learn something new or dive deeper into something your passionate about
+- Learn something new or dive deeper into something you’re passionate about
 - Articulate to someone on a topic so the audience can walk away with a clear understanding of what you are trying to teach/explain/inform/entertain
 
 ## Description of Talk
 - Be creative! This is your chance to show/present to everyone something awesome! It doesn't have to be about code, but it can be.
-- Presentations will be given on thursday and friday. There will be signup sheets posted monday on a "first come, first serve" basis.
+- Presentations will be given on Thursday and Friday. There will be signup sheets posted Monday on a "first come, first serve" basis.
 - Plan on presenting for 12 minutes /w 3 minute Q & A to follow
```
Comment (adambray) [inline readme.md:14]: 12 minutes **or less**?

File: readme.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ On thursday and friday of the coming week(28 May and 29 May), we'll be having Wh
 
 ## Weekend Homework
 - Create a repo for your presentation which will contain everything file based you will use barring external props(which are encouraged!)
-- In the repo have at the least a `readme.md` with learning objectives for your presentation
+- In the repo have a `readme.md` with at least 3 learning objectives for your presentation
 
 ## Additional Homework
 Think about what you want to do for your final project. You will be asked to submit a project idea on monday.
```
Comment (adambray) [inline readme.md:27]: Monday should be Tuesday, right?

Comment (andrewsunglaekim) [inline readme.md:27]: think its the weekend hw no?

Comment (jrhorn424) [inline readme.md:27]: There's no class on Monday.

**Pull request merged**